src/PIL/ImageCms.py

```diff
@@ -256,22 +260,40 @@ def __init__(self, profile: str | SupportsRead[bytes] | core.CmsProfile) -> None
                     profile_bytes_path.decode("ascii")
                 except UnicodeDecodeError:
                     with open(profile, "rb") as f:
-                        self._set(core.profile_frombytes(f.read()))
+                        self.profile = core.profile_frombytes(f.read())
                     return
-            self._set(core.profile_open(profile), profile)
+            self.filename = profile
+            self.profile = core.profile_open(profile)
         elif hasattr(profile, "read"):
-            self._set(core.profile_frombytes(profile.read()))
+            self.profile = core.profile_frombytes(profile.read())
         elif isinstance(profile, core.CmsProfile):
-            self._set(profile)
+            self.profile = profile
         else:
             msg = "Invalid type for Profile"  # type: ignore[unreachable]
             raise TypeError(msg)
 
+    def __getattr__(self, attr: str) -> Any:
+        if attr in ("product_name", "product_info"):
+            deprecate(
+                f"PIL.ImageCmsProfile.{attr}",
+                13,
+                action=(
+                    f"Use PIL.ImageCmsProfile.profile.{attr} instead. "
```

Comment:
```suggestion
                f"ImageCms.ImageCmsProfile.{attr}",
                13,
                action=(
                    f"Use ImageCms.ImageCmsProfile.profile.{attr} instead. "
```
Feel free to argue for `f"PIL.ImageCms.imageCmsProfile.{attr}"` instead if you want, but I think we can all agree that `f"PIL.ImageCmsProfile.{attr}"` is not quite right.

Reply:
Whoops, you're right. Thanks!